src/system/components/data-input/shared/input.scss

```diff
@@ -37,6 +37,7 @@
       opacity: $opacity-disabled;
       pointer-events: none;
       cursor: not-allowed;
+      background-color: $color-neutral-80;
```

Comment:
Please add a new background-color token in `tokens/color.yml` named background-color-disabled and use it here. We should keep the separation between the whole color palette (color choices) and colors for specific use cases (color decisions).